<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EquipWindow</class>
 <widget class="QMainWindow" name="EquipWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1309</width>
    <height>634</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Выбор оснащения</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QFrame" name="frame_current">
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>3</number>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Текущее оснащение противника</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QTableWidget" name="tableWidget_current">
            <property name="statusTip">
             <string>Нажмите два раза для удаления из текущего оснащения противника</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_data">
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="lineWidth">
       <number>3</number>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Возможные варианты оснащения</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QListWidget" name="listWidget_data">
            <property name="statusTip">
             <string>Нажмите для выбора варианта оснащения</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string>Дополнительные параметры</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QGroupBox" name="groupBox_4">
               <property name="title">
                <string>Высота атаки</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <item>
                 <widget class="QComboBox" name="comboBox_alt">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>0</height>
                   </size>
                  </property>
                  <item>
                   <property name="text">
                    <string>Стратосфера</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Большая и средняя, без вертикального маневра</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Большая и средняя, с вертикальным маневром</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_5">
               <property name="title">
                <string>Атака в</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <widget class="QComboBox" name="comboBox_sfer">
                  <item>
                   <property name="text">
                    <string>Заднюю полусферу</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Переднюю полусферу</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <layout class="QGridLayout" name="gridLayout">
               <item row="0" column="0">
                <widget class="QLabel" name="label">
                 <property name="statusTip">
                  <string>Скорость цели</string>
                 </property>
                 <property name="text">
                  <string>Vц, км/ч</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QDoubleSpinBox" name="doubleSpinBox_V">
                 <property name="statusTip">
                  <string>Скорость цели</string>
                 </property>
                 <property name="maximum">
                  <double>100000000000000000000.000000000000000</double>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_2">
                 <property name="statusTip">
                  <string>Исходный рубеж</string>
                 </property>
                 <property name="text">
                  <string>Sисх, км</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QDoubleSpinBox" name="doubleSpinBox_S">
                 <property name="statusTip">
                  <string>Исходный рубеж</string>
                 </property>
                 <property name="maximum">
                  <double>100000000000000000000.000000000000000</double>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_3">
                 <property name="statusTip">
                  <string>В момент ввода в бой необходимо, чтобы цель находилась впереди истребителя на расстоянии Dk</string>
                 </property>
                 <property name="text">
                  <string>Dk, км</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QDoubleSpinBox" name="doubleSpinBox_D">
                 <property name="statusTip">
                  <string>В момент ввода в бой необходимо, чтобы цель находилась впереди истребителя на расстоянии Dk</string>
                 </property>
                 <property name="maximum">
                  <double>100000000000000000000.000000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" name="lay_but_add"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_airbase">
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="lineWidth">
       <number>3</number>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <widget class="QGroupBox" name="groupBox_6">
         <property name="title">
          <string>Свои войска</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Количество аэродромов</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBox_airbase"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTableWidget" name="tableWidget_airbase">
            <property name="statusTip">
             <string>Нажмите два раза для удаления из текущего оснащения противника</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1309</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
